plottable.js

```diff
@@ -2300,20 +2310,31 @@ var XYPlot = (function (_super) {
         _this.addClass("xy-plot");
         _this._adjustYDomainOnChangeFromXCallback = function (scale) { return _this._adjustYDomainOnChangeFromX(); };
         _this._adjustXDomainOnChangeFromYCallback = function (scale) { return _this._adjustXDomainOnChangeFromY(); };
+        // the pixel space X translate that has occurred since the last time we rendered
         var _deltaX = 0;
+        // the pixel space Y translate that has occurred since the last time we rendered
         var _deltaY = 0;
+        // the X scale that has occurred since the last time we rendered
         var _scalingX = 1;
+        // the Y scale that has occurred since the last time we rendered
         var _scalingY = 1;
+        // the X scale's domain the last time we rendered
         var _lastSeenDomainX = [null, null];
+        // the Y scale's domain the last time we rendered
         var _lastSeenDomainY = [null, null];
         var _timeoutReference = 0;
-        var _deferredRenderingTimeout = 500;
+        var _deferredRenderingTimeout = 200;
+        // call this every time the scales change (every pan/zoom event).
+        // this method will "render" now by applying a transform, and then
+        // debounce the true rendering
         var _registerDeferredRendering = function () {
             if (_this._renderArea == null) {
                 return;
             }
-            _this._renderArea.attr("transform", "translate(" + _deltaX + ", " + _deltaY + ")" +
-                "scale(" + _scalingX + ", " + _scalingY + ")");
+            _this._renderArea.attr("transform", "translate(" + _deltaX + ", " + _deltaY + ") scale(" + _scalingX + ", " + _scalingY + ")");
+            if (_this._canvas != null) {
+                _this._canvas.style("transform", "translate(" + _deltaX + "px, " + _deltaY + "px) scale(" + _scalingX + ", " + _scalingY + ")");
+            }
             clearTimeout(_timeoutReference);
             _timeoutReference = setTimeout(function () {
```

Comment:
A destroyed plot will have empty datasets so nothing will be drawn; a detached dataset should (arguably) still draw since it's expected to be reattached to something. I think we're okay here? We already do a `_renderArea == null` check when we register, so I think the worse that happens is we render to a detached element once